CMakeLists.txt

```diff
@@ -461,19 +461,14 @@ if(CMAKE_CXX_COMPILER_ID MATCHES Clang)
 elseif(CMAKE_CXX_COMPILER_ID MATCHES GNU)
 	add_compile_options(
 		-Wall
-		-Wc++11-compat
```

Comment:
the `RC_1_2` branch is still meant to build in C++11 mode, so I think this is a good warning to have.

Reply:
it is already covered by `-Wall`.